aip/general/0140.md

```diff
@@ -88,6 +88,20 @@ For uniformity, field names that contain both a noun and an adjective
 - `collected_items` (**not** `items_collected`)
 - `imported_objects` (**not** `objects_imported`)
 
+### Verbs
+
+Field names **must not** be named to reflect an intent or action. They
+**must not** be verbs. Rather, because the field defines the *desired
+value* for mutations, e.g. Create and Update, and the *current value*
```

Comment:
I think the markdown for italicizing (at least in this "preview" on github) indicates the formatting isn't being applied correctly. Can you just double check that? Might need to close it on line 94 and reopen it on 95.

Reply:
Looks right on the [preview](https://github.com/aip-dev/google.aip.dev/pull/1078/files?short_path=7de9833#diff-7de983345e17e37eb41b1a8d22e476a53d0c0736fd074b7251341bbda2cb08f7), but I'll fire it up on site-generator just the same.

Reply:
Ah ok cool. In code review mode, GH tries to get clever and semi-format the markdown, but it doesn't work correctly
![Screenshot 2023-04-27 at 9 50 27 AM](https://user-images.githubusercontent.com/6644735/234933947-a70fccf8-861c-4733-b786-9f7401ff75e7.png)

See how `value*` isn't italicized, but the following text is, incorrectly? Boo GitHub.

Reply:
Yeah, it's different on src vs preview. :(

Reply:
Verified that it renders correctly with local dev env of site-generator...